src/libcore/convert.rs

```diff
@@ -1,26 +1,25 @@
-//! Traits for conversions between types.
 //!
-//! The traits in this module provide a general way to talk about conversions
-//! from one type to another. They follow the standard Rust conventions of
-//! `as`/`into`/`from`.
+//! The traits in this module provide a way to convert from one type to another type.
 //!
-//! Like many traits, these are often used as bounds for generic functions, to
-//! support arguments of multiple types.
+//! Each trait serves a different purpose:
```

Comment:
... however, we *can* move it to be part of this paragraph though; I think that's where it fits in.